docs/generate-evidence-strings.md

```diff
@@ -113,7 +113,13 @@ After the evidence strings have been generated, summary metrics need to be updat
 ### Submit evidence strings
 The evidence string file (`evidence_strings.json`) must be uploaded to the [Open Targets Google Cloud Storage](https://console.cloud.google.com/storage/browser/otar012-eva/) and be named in the format `cttv012-[yyyy]-[mm]-[dd].json.gz` (e.g. `cttv012-2020-10-21.json.gz`).
 
-More details can be found on [Open Targets Github wiki](https://github.com/opentargets/data_release/wiki/OT006-Data-Submission#ot009-evidence-string-generation-json-schema-validation--submission).
+Once the upload is complete, send an email to Open Targets helpdesk containing the following information:
```

Comment:
Given that it is the **HelpDesk's email address**, I **would add it** (e.g. ``...an email to Open Targets helpdesk (emailaddress [at] ebi.ac.uk) containing...``) so that there are no confusions or mistaken addresses to which this information is sent.